docs/api/Auth0.Core.Exceptions.RateLimitApiException.html

```diff
@@ -89,6 +89,9 @@ <h5>Implements</h5>
   </div>
   <div class="inheritedMembers">
     <h5>Inherited Members</h5>
+    <div>
+      <a class="xref" href="Auth0.Core.Exceptions.ApiException.html#Auth0_Core_Exceptions_ApiException_CreateSpecificExceptionAsync_System_Net_Http_HttpResponseMessage_">ApiException.CreateSpecificExceptionAsync(HttpResponseMessage)</a>
+    </div>
```

Comment:
Docs wasn't regenerated when this change was made https://github.com/auth0/auth0.net/commit/58dc202df1926d90d8b8bbfab050df63cf97e4ec